ddtrace/llmobs/_integrations/base.py

```diff
@@ -99,7 +100,7 @@ def _set_base_span_tags(self, span: Span, **kwargs) -> None:
         """Set default LLM span attributes when possible."""
         pass
 
-    def trace(self, pin: Pin, operation_id: str, **kwargs: Dict[str, Any]) -> Span:
+    def trace(self, pin: Pin, operation_id: str, span_type: Optional[str] = None, **kwargs: Dict[str, Any]) -> Span:
```

Comment:
```suggestion
    def trace(self, pin: Pin, operation_id: str, submit_to_llmobs: bool = False, **kwargs: Dict[str, Any]) -> Span:
```
Sorry for last suggestion, I think it makes the most sense if we just mark this as a boolean flag instead of having to specify the LLMObs span type, since we don't really care about other span types